src/zip317.cpp

```diff
@@ -14,15 +14,22 @@ CAmount CalculateConventionalFee(size_t logicalActionCount) {
     return MARGINAL_FEE * std::max(GRACE_ACTIONS, logicalActionCount);
 }
 
+template<typename T>
+size_t GetUTXOFieldSize(const std::vector<T>& utxos) {
+    auto size = GetSerializeSize(utxos, SER_NETWORK, PROTOCOL_VERSION);
+    auto countSize = GetSizeOfCompactSize(utxos.size());
```

Comment:
This is misnamed because it is counting either inputs or outputs, not UTXOs.
```suggestion
size_t GetTxInOutFieldSize(const std::vector<T>& txins_or_txouts) {
    auto size = GetSerializeSize(txins_or_txouts, SER_NETWORK, PROTOCOL_VERSION);
    auto countSize = GetSizeOfCompactSize(txins_or_txouts.size());
```

Reply:
I keep wanting a word that means “in or out”, the way that “(logical) action” means “in and out”. Could we use “port” for that? So, `size_t GetTxPortFieldSize(const std::vector<T>& txports)`?

Reply:
"Port" is already overloaded. It's very common to use "I/O" or "IO"; I'll update the suggestion with that.